packages/babel-helpers/scripts/generate-helpers.js

```diff
@@ -38,31 +45,19 @@ import template from "@babel/template";
     }
     const { minVersion } = minVersionMatch.groups;
 
-    // TODO: We can minify the helpers in production
-    const source = fileContents
-      // Remove comments
-      .replace(/\/\*[^]*?\*\/|\/\/.*/g, "")
-      // Remove multiple newlines
-      .replace(/\n{2,}/g, "\n");
+    const source = await minify(fileContents, {
+      compress: false,
```

Comment:
Does the default compressor option break babel helper? May leave a note/todo here.

I think we disable `mangle` because babel helper does not ship with a source map.

Reply:
`compress: true` works too, I disabled it only to keep the code as is (compress rewrites some expressions, moves declarations and conditionals into for loop initialization, etc.); if readability is not a concern here, it can be enabled.

Reply:
As long as debuggability is not affected too much (I want to be able to understand stack traces in bug reports), that's ok.

Reply:
Yeah the stack traces will be fine since we have disabled `mangle`, which renames identifiers to the shortest available.